cristin-import/src/main/java/no/unit/nva/cristin/mapper/JournalBuilder.java

```diff
@@ -94,15 +94,15 @@ private PublicationInstance<? extends Pages> createJournalArticle() {
 
         var secondaryCategory = getCristinObject().getSecondaryCategory();
         if (CristinSecondaryCategory.JOURNAL_ARTICLE.equals(secondaryCategory)) {
-            return new ProfessionalArticle(numberOfPages, extractVolume(), extractIssue(), null);
+            return new ProfessionalArticle(numberOfPages, extractVolume(), extractIssue(), extractArticleNumber());
```

Comment:
Can null pointers occur in the extractArticleNumber() method?

Reply:
Wrapping in optional of nullable